config/config.js

```diff
@@ -68,10 +69,20 @@ var validateEnvironmentVariable = function () {
   console.log(chalk.white(''));
 };
 
-/** Validate config.domain is set
+/**
+ * Validate user is running supported Node.js version
+ */
+var validateNodeJsVersion = function (config) {
+  if (!semver.satisfies(process.version, config.meanjs.engines.node)) {
+    console.log(chalk.red('+ Important warning: your version of Node (' + process.version.substr(1) + ') is unsupported by MEAN.js. Please use a version that satisfies the following requirements: ' + config.meanjs.engines.node + ', as specified in the package.json file.'));
+  }
+};
+
+/**
+ * Validate config.domain is set
  */
 var validateDomainIsSet = function (config) {
-  if (!config.domain) {
+  if (!config.domain && process.env.NODE_ENV === 'production') {
```

Comment:
My rationale behind the change was the following:
- Seeing many people install new software, getting a red error message about configuration post-install is unexpected - the first thought is that something went wrong during the installation process, probably with Node.
- By the time users actually deploy, I would expect them to have run the app several times in production mode in a safe environment to check it: providing multiple opportunities to see the message.

Anyway, I can see where you're coming from @mleanos. I think what we really need is just a better and more descriptive message. What do you mean by _scary_ implications?. I believe you might be best placed to write the message ;). I'll remove the change for now.